tests/integ/scala/test_function_suite.py

```diff
@@ -3779,6 +4087,10 @@ def test_get_path(session, v, k):
     )
 
 
+@pytest.mark.skipif(
+    "config.getvalue('local_testing_mode')",
+    reason="get is not yet supported in local testing mode.",
```

Comment:
I remember you added support to `get` in the PR: https://github.com/snowflakedb/snowpark-python/pull/1431